install/kubernetes/cilium/values.yaml

```
@@ -2029,6 +2042,15 @@ operator:
       relabelings: ~
       # -- Metrics relabeling configs for the ServiceMonitor cilium-operator
       metricRelabelings: ~
+  dashboards:
+    # -- Grafana dashboards for cillium-operator
```

More typos (sorry I didn't catch this yesterday):

```suggestion
    # -- Grafana dashboards for cilium-operator
```